addons/website/views/snippets/snippets.xml

```diff
@@ -1114,6 +1114,20 @@
                        data-img="/website/static/src/img/snippets_options/header_template_sidebar.svg"/>
```

Comment:
Commit message detail: try to have a title that complete the sentence "When applied, this commit will ...". So here, "When applied, this commit will header container width" is not good :+1: